test/markup/swift/keywords.txt

```diff
@@ -4,6 +4,11 @@ String.Type Type
 String.init init
 String.self self
 
+class
```

Comment:
You can't do this with class because it expects the next word after it to be a title... so you need to at least have `class Blah {}` or something.

Reply:
Same with the others that have context (like struct, etc).

Reply:
I would remove this test and add an entirely separate file for actors probably.

Reply:
I've created a separate suite for all type definitions as they all take a similar form at the moment, could be broken up for each individual type definition keyword later?